<ui version="4.0" >
 <class>CreateShortcutCommandWidget</class>
 <widget class="QWidget" name="CreateShortcutCommandWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Shortcut:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KKeySequenceWidget" native="1" name="ksShortcut" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lbSpecialShortcut" >
     <property name="text" >
      <string>Special Shortcut:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="KComboBox" name="cbSpecialShortcut" >
       <item>
        <property name="text" >
         <string>Alt+Tab</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Alt+F4</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ctrl+Esc</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Ctrl+Alt+Esc</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="pbApplySpecialShortcut" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>kkeysequencewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header location="global" >kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header location="global" >kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
